api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/impac/tenant_dashboards_controller.rb

```diff
@@ -0,0 +1,91 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::Admin::Impac::TenantDashboardsController
+  extend ActiveSupport::Concern
+
+  #==================================================================
+  # Included methods
+  #==================================================================
+  # 'included do' causes the included code to be evaluated in the
+  # context where it is included rather than being executed in the module's context
+  included do
+    respond_to :json
+  end
+
+  DASHBOARD_DEPENDENCIES = [:widgets, :'widgets.kpis', :kpis, :'kpis.alerts', :'kpis.alerts.recipients']
+
+  #==================================================================
+  # Instance methods
+  #==================================================================
+  # GET /mnoe/jpi/v1/admin/impac/tenant_dashboards
+  def index
+    dashboards
+    render template: 'mno_enterprise/jpi/v1/impac/dashboards/index'
+  end
+
+  # POST /mnoe/jpi/v1/admin/impac/tenant_dashboards
+  #   -> POST /api/mnoe/v1/users/1/dashboards
+  def create
+    @dashboard = MnoEnterprise::Dashboard.create!(dashboard_create_params)
+    MnoEnterprise::EventLogger.info('dashboard_create', tenant.id, 'Dashboard Creation', @dashboard)
```

Comment:
here it should still be the current_user id, we are logging the actions of users.